integration/lifecycle_test.go

```diff
@@ -75,13 +76,37 @@ var _ = Describe("Lifecycle", func() {
 								return err
 							}
 
-							if err := container.CopyDirToContainer(ctx, "workspace", "/home/ubuntu/", 0o755); err != nil {
+							if err := container.CopyDirToContainer(ctx, "./testdata/workspace", "/home/ubuntu/", 0o755); err != nil {
 								return err
 							}
 
 							code, _, err := container.Exec(ctx, []string{"chown", "-R", "ubuntu:ubuntu", "/home/ubuntu/workspace"})
 							if code != 0 {
-								return fmt.Errorf("failed to run run chown, RC: %d", code)
+								return fmt.Errorf("failed to run chown, RC: %d", code)
```

Comment:
Is it worth to have a helper method `runInContainer(ctx, container, cmd) error`? To have less duplication (e.g. if code != 0). I will try out how it looks.